src/storage.js

```diff
@@ -198,14 +198,17 @@ export class LocalStorageStore {
     }
   }
 
-  cleanWallet() {
-    this.removeItem(IS_HARDWARE_KEY);
-    this.removeItem(CLOSED_KEY);
+  cleanAccessData() {
```

Comment:
suggestion: Could you add descriptions for these methods? It's not clear to me when we should use `clearWallet` or `cleanAccessData`.

Reply:
Done